src/game/control.c

```diff
@@ -215,6 +215,9 @@ int32_t ControlPhase(int32_t nframes, GAMEFLOW_LEVEL_TYPE level_type)
     while (m_FrameCount >= 0) {
         CheckCheatMode();
         if (g_LevelComplete) {
+            if (g_Config.disable_healing_between_levels) {
+                g_LevelStartLaraHealth = g_StoredLaraHealth;
+            }
```

Comment:
Is this a relic of the old method? The one thing I wasn't sure about. The testing I did works, but maybe I should remove this and test again.